public/video-ui/src/actions/actions.ts

```diff
@@ -11,22 +8,7 @@ type ShowError = BaseAction<'SHOW_ERROR'> & {
     error: unknown
 }
 
-type AddProjectRequest = BaseAction<'ADD_PROJECT_REQUEST'> & {
-}
-
-type AddProjectReceive = BaseAction<'ADD_PROJECT_RECEIVE'> & {
-    video: Video,
-}
 
-/**
- * A union of the Action types that dispatched from ts files.
- *
- * This is not yet a complete list of all the actions the application handles as
- * some actions dispatched from js files.
- */
-type KnownAction =
-    ShowError |
-    AddProjectRequest |
-    AddProjectReceive;
+type KnownAction = ShowError
```

Comment:
It should be possible to remove this file after #1259 is also merged. RTK will provide us typing for our actions, so this should be unnecessary.